src/order/category/CompleteLattice.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.BoundedLattice
+import order.hom.complete_lattice
+
+/-!
+# The category of complete lattices
+
+This file defines `CompleteLattice`, the category of complete lattices.
+-/
+
+universes u
+
+open category_theory
+
+/-- The category of complete lattices. -/
+def CompleteLattice := bundled complete_lattice
+
+namespace CompleteLattice
+
+instance : has_coe_to_sort CompleteLattice Type* := bundled.has_coe_to_sort
+instance (X : CompleteLattice) : complete_lattice X := X.str
+
+/-- Construct a bundled `CompleteLattice` from a `complete_lattice`. -/
+def of (α : Type*) [complete_lattice α] : CompleteLattice := bundled.of α
+
+instance : inhabited CompleteLattice := ⟨of punit⟩
+
+instance : bundled_hom @complete_lattice_hom :=
```

Comment:
This reminded me: should `bundled_hom` become a subclass of `fun_like`?

Reply:
I think this is a good question for @semorrison

Reply:
How so? This is a technical class used for tagging types of homs when constructing a category. It's not meant to be used back in the non category theory world. As an aside, it doesn't have the correct type signature.

Reply:
Yeah, I'm not sure what this would mean either.